mne/decoding/receptive_field.py

```diff
@@ -0,0 +1,209 @@
+import numpy as np
+import matplotlib.pyplot as plt
+from sklearn.linear_model import Ridge
+from .base import get_coefs
+
+
+class ReceptiveField(object):
+    """Fit a receptive field model.
+
+    Parameters
+    ----------
+    stim_features : array, shape (n_features,)
+        Names for the input features to the model.
+    lags : array, shape (n_lags)
+        Time lags to be included in the model. This will be combined
+        with `sfreq` to convert these values into indices in the data.
```

Comment:
Not sure on this one...I think I've used them before but lemme know and I can take them out